charts/mediawiki/Chart.yaml

```diff
@@ -1,8 +1,7 @@
 apiVersion: v1
-appVersion: "1.37"
```

Comment:
Considering `appVersion` is optional and never used after applying the changes from this PR, I would think it's better to remove it.